### PR TITLE
Add Property class and its identity and data classes

### LINKED ISSUE #43: Property classes have not been created

### PR DESCRIPTION
**Pull request by lkwlkww**
close #43

### CODE REVIEW

**Review by yuehernkang — APPROVED** [review]
LGTM

File: src/main/java/seedu/condonery/model/property/Tag.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package seedu.condonery.model.property;
```
Comment (misterpuffin) [inline src/main/java/seedu/condonery/model/property/Tag.java:1]: doesn't seem like this file is being used! is this a duplicate of `model/tag/Tag`?

**Review by misterpuffin — COMMENTED** [review]
Other than that, looks good to me! Just need to confirm that the Tag file is not duplicated

File: src/main/java/seedu/condonery/model/property/Tag.java (hunk 1 above)
Comment (lkwlkww) [reply src/main/java/seedu/condonery/model/property/Tag.java:1]: Thanks Haoren! I wanted to delete it but forgot

**Review by zacchaeuschok — APPROVED** [review]

**Pull request merged**